vyxal/elements.py

```diff
@@ -5572,6 +5572,16 @@ def separate_runl_encode(lhs, ctx: Context):
     return lengths
 
 
+def mod_pow(lhs, rhs, other, ctx: Context):
+    """Element ∆%
+    (any, any, any) -> a ** b mod c
+    """
+    ts = vy_type(lhs, rhs, other, simple=True)
+    if list in ts:
+        return vectorise(mod_pow, lhs, rhs, other, ctx=ctx)
+    return pow(int(lhs), int(rhs), int(other))
```

Comment:
Make sure this is converted to a Sympy type - raw python ints will confuse the printing and typing system

Reply:
```suggestion
    return sympy.nsimplify(pow(int(lhs), int(rhs), int(other)), rational=True)
```